src/styles/blocks/page.scss

```diff
@@ -0,0 +1,7 @@
+.page {
+  font-family: Roboto, sans-serif;
+
+  &__body {
+    margin: 0;
```

Comment:
your margins are reseted in normalize.scss. And I'm not sure where do you use this class. The same goes to page.scss. Seems to be redundant file